addons/modules/config.cpp

```diff
@@ -0,0 +1,16 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        units[] = {"CBA_ModuleAttack","CBA_ModuleDefend","CBA_ModulePatrol"};
+        requiredVersion = 0.1;
+        requiredAddons[] = {"A3_Modules_F"};
+        author = "$STR_CBA_Author";
+        authors[] = {"WiredTiger"};
+        authorUrl = "";
```

Comment:
I'm making the change now. I just took a quick look over everything else and to me it looks good, but is there anything else that needs changed to be in line with CBA standards?